crates/edr_napi/npm/linux-x64-musl/package.json

```diff
@@ -4,7 +4,7 @@
     "url": "https://github.com/NomicFoundation/edr.git",
     "type": "git"
   },
-  "version": "0.2.0-alpha.1",
+  "version": "0.11.3",
```

Comment:
Do you know why these weren't up to date?

Reply:
I think the automated release process only bumps the package version in the root package.json in `crates/edr_napi`.